opta/utils/__init__.py

```diff
@@ -21,6 +21,40 @@
 yaml = YAML(typ="safe")
 
 
+class SensitiveFormatter(Formatter):
+    regex_and_replacements: List[Tuple[str, str]] = [
+        (
+            r"([\"\: \'\n])[0-9]{12}([\"\: \'\n])",
+            "\g<1>REDACTED\g<2>",
+        ),  # An AWS Account ID is always exactly 12 digits
+        (
+            r"([\"\: \'\n]?)project([\"\: \'\n]?): ([\"\: \'\n]?)[a-zA-Z0-9_\-]+([\"\: \'\n]?)",
+            "\g<1>project\g<2>: \g<3>REDACTED\g<4>",
+        ),
+        (r"projects/[a-zA-Z0-9_\-]+/", "projects/REDACTED/"),
+        (
+            r"@[a-zA-Z0-9_\-]+.iam.gserviceaccount.com",
+            "@REDACTED.iam.gserviceaccount.com",
+        ),
+        (r"[a-zA-Z0-9_\-]+.svc.id.goog", "REDACTED.svc.id.goog",),
+        (r"gcr.io/[a-zA-Z0-9_\-]+", "gcr.io/REDACTED"),
```

Comment:
```suggestion
        (
            r"@[a-zA-Z0-9_\-]+\.iam\.gserviceaccount\.com",
            "@REDACTED.iam.gserviceaccount.com",
        ),
        (r"[a-zA-Z0-9_\-]+\.svc\.id\.goog", "REDACTED.svc.id.goog",),
        (r"gcr\.io/[a-zA-Z0-9_\-]+", "gcr.io/REDACTED"),
```

Treat the `.` in domain names as literals.